Fst2List.cpp

```diff
@@ -1934,8 +1936,8 @@ int CFstApp::outWordsOfGraph(int depth) {
   Fst2Tag Tag;
   unichar *suffixPtr;
   unichar *wordPtr;
-  unichar *ep; //input buffer ptr
-  unichar *tp; //transducer buffer ptr
+  unichar *ibp; //input buffer ptr
+  unichar *obp; //transducer buffer ptr
```

Comment:
```suggestion
    unichar *outputBufferPtr  // transducer buffer ptr
```

Reply:
Giving sensible names to types and variables is much better than using obscure names that you must then explain through comments.